LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015
+Copyright (c) 2018
```

Comment:
should probably be `2015-2018`?

Reply:
The upgrade changed it to 2018. Should it be 2015-2018?